common/config/rush/common-versions.json

```diff
@@ -71,12 +71,13 @@
     "@azure/ms-rest-nodeauth": ["^0.9.2"],
     // Idenity is moving from v1 to v2. Moving all packages to v2 is going to take a bit of time, in the mean time we could use v2 on the perf-identity tests.
     "@azure/identity": ["^2.0.0-beta.4", "2.0.0-beta.3", "^1.1.0"],
-    // App Config uses keyvault-secrets in a sample, switch to latest once the preview becomes GA
     // Issue #14771 tracks updating to these versions
     "@microsoft/api-extractor": ["7.13.2"],
     "prettier": ["2.2.1"],
     // All packages should move to 1.0.0 once core-rest-pipeline 1.1.0 GAs
-    "@azure/core-tracing": ["1.0.0-preview.11"]
+    "@azure/core-tracing": ["1.0.0-preview.11"],
+    // @azure/core-lro 1.0.6 removed a dependency on @azure/core-http which allows KeyVault Admin to also remove that dependency.
+    "@azure/core-lro": ["^1.0.6"]
```

Comment:
I feel like we should bump this everywhere

Reply:
Sure, done!